projects/components/src/lib/datetime/datetime.component.html

```diff
@@ -0,0 +1,9 @@
+<div>
+  <input matInput [value]="model" (change)="updateDate($any($event).target.value)"
```

Comment:
do we need to wrap this in a label tag like other inputs?

Reply:
we don't.